src/test/resources/fixtures/updatecase/amend-divorce-details-d81-joint.json

```diff
@@ -1,409 +1,410 @@
 {
-  "token": "test_token",
-  "event_id": "FR_issueApplication",
-  "case_details": {
-    "id": "12345678",
-    "jurisdiction": "divorce",
-    "state": "caseAdded",
-    "case_type_id": "FinancialRemedyMVP2",
-    "case_data": {
-      "updateIncludesRepresentativeChange": "No",
-      "PBANumber": "PBA123456",
-      "d81Question": "Yes",
-      "PBAreference": "ABCD",
-      "consentOrder": {
-        "document_url": "http://file1",
-        "document_filename": "file1",
-        "document_binary_url": "http://file1.binary"
+  "token" : "test_token",
+  "event_id" : "FR_issueApplication",
+  "case_details" : {
+    "id" : "12345678",
+    "jurisdiction" : "divorce",
+    "state" : "caseAdded",
+    "case_type_id" : "FinancialRemedyMVP2",
+    "case_data" : {
+      "updateIncludesRepresentativeChange" : "No",
+      "PBANumber" : "PBA123456",
+      "d81Question" : "Yes",
+      "PBAreference" : "ABCD",
+      "consentOrder" : {
+        "document_url" : "http://file1",
+        "document_filename" : "file1",
+        "document_binary_url" : "http://file1.binary"
       },
-      "d81Joint": {
-        "document_url": "http://file1",
-        "document_filename": "file1",
-        "document_binary_url": "http://file1.binary"
+      "d81Joint" : {
+        "document_url" : "http://file1",
+        "document_filename" : "file1",
+        "document_binary_url" : "http://file1.binary"
       },
-      "d81Applicant": {
-        "document_url": "http://file1",
-        "document_filename": "file1",
-        "document_binary_url": "http://file1.binary"
+      "d81Applicant" : {
+        "document_url" : "http://file1",
+        "document_filename" : "file1",
+        "document_binary_url" : "http://file1.binary"
       },
-      "d81Respondent": {
-        "document_url": "http://file1",
-        "document_filename": "file1",
-        "document_binary_url": "http://file1.binary"
+      "d81Respondent" : {
+        "document_url" : "http://file1",
+        "document_filename" : "file1",
+        "document_binary_url" : "http://file1.binary"
       },
-      "solicitorFirm": "FirmA",
-      "solicitorName": "Solictor",
-      "applicantLName": "Guy",
-      "authorisation3": "2010-01-01",
-      "solicitorEmail": "test@admin.com",
-      "solicitorPhone": "1234",
-      "applicantFMName": "Poor",
-      "authorisation2b": "test",
-      "otherCollection": [
+      "solicitorFirm" : "FirmA",
+      "solicitorName" : "Solictor",
+      "applicantLName" : "Guy",
+      "authorisation3" : "2010-01-01",
+      "solicitorEmail" : "test@admin.com",
+      "solicitorPhone" : "1234",
+      "applicantFMName" : "Poor",
+      "applicantRepresented" : "Yes",
```

Comment:
The only changes that I've made in these test json files is this property. This property was missing in these test data files. We have data for the solicitor but for some reason we didn't have a property to show that the applicant is represented.